values.yaml

```diff
@@ -788,6 +792,7 @@ redis:
     existingSecret: ""
   ## Additional deployment annotations
   podAnnotations: {}
+  svcAnnotations: {}
```

Comment:
why here it is `svcAnnotations` while the others are `serviceAnnotations`?

Reply:
@zyyw Oh yeah my bad. I firstly did `svcAnnotations` and then I realized that it was already done on core and notary component as `serviceAnnotations` so I've adapted the syntax and missed one ^^'